docker/dev/django/Dockerfile

```
@@ -56,3 +56,4 @@ ENTRYPOINT ["/entrypoint.sh"]
 # Don't use root as default user
 USER $APP_USER
 WORKDIR $APP_DIR
+RUN mkdir -p .cache
```

Je le rajouterais bien en tant que `ARG` pour pouvoir faire ceci :
```suggestion
RUN mkdir -p $XDG_CACHE_HOME
```
Comme ça c'est docker compose qui contrôle tout et on éclate pas l'info à plusieurs endroits.

A vérifier, mais si les `ARG` restent dans le docker alors il n'y aurais même plus besoin de l'ajouter en tant qu'`ENV` en plus, ou alors via un `ENV XDG_CACHE_HOME $XDG_CACHE_HOME` peut_être :shrug:.